tutorials/in-app/plinkoMultiplier.json

```diff
@@ -0,0 +1,883 @@
+          }
+        },
+        "placement": "left"
+      },
+      "isOnClosableDialog": true
+    },
+    {
+      "elementToHighlightId": "#instruction-editor-dialog #ok-button",
+      "nextStepTrigger": {
+        "absenceOfElement": "#instruction-editor-dialog"
+      },
+      "tooltip": {
+        "description": {
+          "messageByLocale": {
+            "en": "Nice! our **$(scoreMultiplier)** will now be updated thanks to the **Multiplier variable** that we converted to a text!"
+          }
+        },
+        "placement": "top"
+      }
+    },
+    {
+      "elementToHighlightId": "[id=\"**Score multiplier-unfolded\"]",
+      "nextStepTrigger": {
+        "presenceOfElement": "[id=\"**Score multiplier-folded\"]"
+      },
+      "tooltip": {
+        "description": {
+          "messageByLocale": {
+            "en": "Finally, we need to take our multiplier into account when calculating the score. Let's **fold the Score multiplier** group with the arrow on the left."
+          }
+        },
+        "placement": "bottom"
+      }
+    },
+    {
+      "elementToHighlightId": "[id=\"**Hitting pegs-folded\"]",
+      "nextStepTrigger": {
+        "presenceOfElement": "[id=\"**Hitting pegs-unfolded\"]"
+      },
+      "tooltip": {
+        "description": {
+          "messageByLocale": {
+            "en": "And let's **unfold the Hitting pegs** group with the arrow on the left."
+          }
+        },
+        "placement": "bottom"
+      }
+    },
+    {
+      "elementToHighlightId": "#event-3-action-0",
+      "nextStepTrigger": {
+        "presenceOfElement": "#instruction-parameters-container"
+      },
+      "tooltip": {
+        "description": {
+          "messageByLocale": {
+            "en": "This first event manages the logic when the player hits a peg. Let's modify the score action by **double-clicking on it**!"
+          }
+        }
+      }
+    },
+    {
+      "elementToHighlightId": "#parameter-2-expression-field",
+      "nextStepTrigger": {
+        "valueEquals": "1*Variable(Multiplier)"
+      },
+      "tooltip": {
+        "description": {
+          "messageByLocale": {
+            "en": "We want to multiply the score added by the multiplier, so let's enter `1*Variable(Multiplier)`."
+          }
+        },
+        "placement": "left"
+      },
+      "isOnClosableDialog": true
+    },
+    {
+      "elementToHighlightId": "#instruction-editor-dialog #ok-button",
+      "nextStepTrigger": {
+        "absenceOfElement": "#instruction-editor-dialog"
+      },
+      "tooltip": {
+        "description": {
+          "messageByLocale": {
+            "en": "This will make our score update according to the multiplier! Let's save."
+          }
+        },
+        "placement": "top"
+      }
+    },
+    {
+      "elementToHighlightId": "#event-7-action-0",
```

Comment:
I understand why you did that. I thought about it but the fact that one that reorders the events breaks the whole thing felt a bit dangerous to me

Reply:
We could: 
- either add a way to "tag" events from the IDE (as an advanced option that allows to add a "uniqueIdentifierForTutorials" to an event/action/condition) 
- or keep this approach and disable drag'n'drop/adding new events/etc in tutorials.

Reply:
I've reworked the id-ing slightly so that it's less error-prone. (But still if you drag/drop add a whole unexpected new event)

I suggest we monitor the completion rate, and the fall off, to possibly prevent actions inside the mini tuto.